adagrams/game.py

```diff
@@ -1,11 +1,96 @@
+import random
+import copy
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+        1: ["A", "E", "I", "O", "U", "N", "R", "S", "T"],
+        2: ["D", "G"],
+        3: ["B", "C", "M", "P"],
+        4: ["F", "H", "V", "W", "Y"],
+        5: ["K"],
+        8: ["J", "X"],
+        10: ["Q", "Z"]
+    }
+
+
+
 def draw_letters():
-    pass
+    letters_for_user = []
+    fresh_copy = copy.copy(LETTER_POOL)
+    while len(letters_for_user) < 10:
+        letter = random.choice(list(fresh_copy.keys()))
```

Comment:
From a strict statistical perspective, when you create a list from fresh_copy.keys(), your draw_hands it doesn't quite follow the distribution of available tiles because you get a list of 26 letters (A-Z).

How could you refactor this logic in draw_letters so that when you draw a letter to build up your hand, you're drawing from a pool with 9 As, 2 B's, 2 C's, etc...?